### ISSUE TITLE
Elevated circulating globotriaosylsphingosine concentration

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11696]
**Preferred term label:**
Elevated circulating globotriaosylsphingosine concentration

**Synonyms**
Lyso-GL3
Lyso-Gb3

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0004343

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fabry  https://link.springer.com/article/10.1186/s13023-025-03922-x

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this is a synonym:
[Elevated circulating lyso-globotriaosylsphingosine concentration HP:0032657](https://hpo.jax.org/browse/term/HP:0032657)
I am not sure if it is completely correct to leave off the "lyso"?

**Comment by MickeySegal** [comment]
Google's AI says:
"Lyso-globotriaosylsphingosine" is just a longer, more explicit chemical descriptor that includes the "lyso-" prefix redundantly, since globotriaosylsphingosine is already the deacylated (lyso) form of globotriaosylceramide (Gb₃).

That fits with the usage in the OJRD paper.

**Comment by kanems** [comment]
FYI PubChem is another good resource to check on compound names https://pubchem.ncbi.nlm.nih.gov/compound/6449939#section=Synonyms

**Comment by pnrobinson** [comment]
@MickeySegal  @kanems 
Following PubChem, Let's make the primary label be

Elevated circulating globotriaosyl lysosphingolipid concentration

I added the other synonyms

thanks
